packages/urlstate/encoder/encoder.test.ts

```
@@ -7,45 +7,51 @@ import {
         expect(encode(obj1)).toEqual(
           enc1,
         );
       })
-      it('decode', () => {
-        const obj = clone(obj1)
+
+      test('decode', () => {
+        const obj = { ...obj1 }
         delete obj.und
-        expect(obj).toEqual(decode(enc1))
+        const decoded = decode<typeof obj>(enc1)
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)